core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -492,12 +508,38 @@ class WorkflowResource {
       }
     }
 
+    // Apply creation_time date filter
+    val creationDateFilter = getCreationDateFilter(creationStartDate, creationEndDate)
+    // Apply lastModified_time date filter
+    val modifiedDateFilter = getModifiedDateFilter(modifiedStartDate, modifiedEndDate)
+    // Apply workflowID filter
+    val workflowIdFilter = getWorkflowIdFilter(workflowIDs)
+    // Apply owner filter
+    val ownerFilter = getOwnerFilter(owners)
+    // Apply operators filter
+    val operatorsFilter = getOperatorsFilter(operators)
+    // Apply projectId filter
+    val projectIdFilter = getProjectFilter(projectIds)
+
+    // combine all filters with AND
+    var optionalFilters: Condition = noCondition()
+    optionalFilters = optionalFilters
+      .and(creationDateFilter)
```

Comment:
It'd save a couple of lines (without loss of readability, I hope) if we just call the get...filter() function right here.

Reply:
done